spanner_orm/tests/migrations_test.py

```diff
@@ -25,11 +28,13 @@
 
 
 class MigrationsTest(unittest.TestCase):
-  TEST_DIR = os.path.dirname(__file__)
+  TEST_DIR = tempfile.mkdtemp()
```

Comment:
it looks like we never clean up the temp directory, should probably do so in a tearDownClass

Reply:
done.